components/ui/forms/__tests__/input-textarea.test.tsx

```diff
@@ -1,15 +1,10 @@
-import { render } from "@testing-library/react";
+import { render, screen } from "@testing-library/react";
 import { InputTextarea } from "../input-textarea";
 
 describe("InputText", () => {
   it("renders correctly", () => {
-    const { container } = render(<InputTextarea />);
+    render(<InputTextarea />);
 
-    expect(container.firstChild).toMatchInlineSnapshot(`
-      <textarea
-        class="shadow-sm focus:ring-blue-500 focus:border-blue-500 block w-full text-sm border-gray-300 rounded-md"
-        rows="4"
-      />
-    `);
+    expect(screen.getByRole("textbox")).toBeInTheDocument();
```

Comment:
I think we can keep the .toMatchInlineSnapshot() usage here since we're actually testing the styling for this component and the snapshot result is very small and readable.